### ISSUE TITLE
Accessibility Issues with month and year picker

### ISSUE THREAD

**Issue by nikeeshi** [hypeserver/react-date-range#416]
## Subject of the issue
The month and year picker do not follow WCAG, violating the rule "Form elements must have lables"
 
## [BUG] Bug Reproduce Steps
1. Install Axe.
https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd
2. Run the test in the demo page. https://hypeserver.github.io/react-date-range/
Instead of this step, you can also run yarn start locally in this repository.
3. Axe reports issues of the month and year picker that "Form elements must have labels".

## [BUG] Expected behaviour
No issue should be reported.

The react-date-range picker should include in their API a way to set a aria-label to the month and year picker.

## Environment
Package Version: - (The current version on Oct 14th in demo page.)
React version: -
Node version: -
Browser: Chrome 85

<img width="967" alt="month" src="https://user-images.githubusercontent.com/2003011/95949315-5c500f80-0e2d-11eb-8cbe-8a98c360ebbd.PNG">

**Comment by kamyar** [comment]
Should be fixed please feel free to open a new issue if something is missing
